helm/ph-ee-fin-front-end /Chart.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v2
```

Comment:
do we have an additional character at the end of the name of the directory   `ph-ee-fin-front-end` in  `helm/ph-ee-fin-front-end /`? This will be difficult to maintain and debug